util/msys2_install.sh

```diff
@@ -43,17 +44,10 @@ if [ -f "FlipInstaller.exe" ]; then
 fi
 
 if [ ! -d "$avrtools" ]; then
-    while true; do
-        echo
-        echo "The AVR toolchain is not installed."
-        echo "This is needed for building AVR based keboards."
-        read -p "Do you want to install it? (Y/N) " res
-        case $res in
-            [Yy]* ) install_avr; break;;
-            [Nn]* ) break;;
-            * ) echo "Invalid answer";;
-        esac
-    done
```

Comment:
I found being able to skip the toolchain installations very useful when I was debugging other parts of this process (like qmk_driver_installer dropping to the command prompt). And the ability to wipe out the current toolchain installation and start again might come in handy. So I'm not sure about blindly running these only if they're not present.

Reply:
Well, simply creating the folder will cause it to create the prompt. 

But a big part of the goal for this PR is to reduce the number of prompts for users, and streamline the install.

Reply:
Maybe having optional arguments for skipping sections?

Reply:
Well, if it is, it would ideally be called from qmk_install.sh, and passed along. 

And ... I'm not exactly sure how to do this in the first place (I haven't even looked yet, though)

Reply:
Yeah, I'm thinking something like `./util/qmk_install.sh --advanced` which will prompt before doing *anything*, whereas without the flag it will install everything.
Making these install scripts more modular/unified/consistent/etc has been on my wishlist for a while, but I guess I'm fine with this for now.

Reply:
That would be really nice, especially for debugging stuff, actually!